### DISCUSSION TITLE
The observation period for a (cancer) registry

### DISCUSSION THREAD

**Discussion by peterprinsen-iknl** [OHDSI/OncologyWG#493] [General]
We have been discussing the observation period among several European cancer registries and we have not yet reached a consensus. We therefor thought it may be useful to continue the discussion on this forum to collect other opinions. Below I will discuss the choices for the start date and the end date that we have come up with so far and their pros and cons.

**OBSERVATION_PERIOD.observation_period_start_date**

**1. The incidence date**

_pros:_

- This is usually the date on which a cancer registry starts collecting information about the patient so this seems a natural choice. It does not need to be the exact incidence date: if the cancer registry collects data on diagnostic tests before the actual diagnosis date the start date may be on the earliest date of such a test.

_cons:_

- We have seen that in the PIONEER studies a common requirement is that there is at least a year of continuous observation before the cohort entry date, which is the diagnosis/incidence date (is this an exception or is this a common requirement in studies?). In these particular studies this was done to check that the patient had no diagnosis of the same disease (prostate cancer) or was not treated for the same disease in the year before. This is necessary because in some data sets, for example insurance claims data, the date of first diagnosis of the disease is not as clear as in a cancer registry so the cohort construction, which should work for all types of data sets, is more complicated. In this particular case it means cancer registries cannot participate in the stuy or the study protocols need to be changed for cancer registries (even though in this particular example we know the patient was not diagnosed with prostate cancer or was treated for prostate cancer in the year before because otherwise there would be another registration in the cancer registry and the second registration (i.e. pertaining to the current incidence date) would only exist if the two cancer diagnoses were two separate primary tumor diagnoses, which is unlikely for prostate cancer). 

**2. The date of birth or the date from which there is full coverage of the region by the registry** 

_pros:_

- Many cancer registries register all primary incidences of cancer in a certain region. This means they follow the people in that region from birth or, if regional coverage is only guaranteed after a certain date, from that date onwards. For example, cancer incidence has been collected in the Netherlands for many years but the Netherlands Cancer Registry only reached full national coverage in 1989. In that case the choice for the start date would be max(1-1-1989, birth date).

_cons:_

- People may leave the region or enter the region (emigration/immigration) at a certain time and this is usually not registered. If a patient is diagnosed with cancer while being outside the region this may not be registered by the cancer registry. This is (probably) mainly an issue in smaller regions: it occurs in larger regions as well but the percentage of people moving in and out is probably lower.
- This may imply that all persons in the region are included in the registry, also the ones that have never been diagnosed with cancer, which is not the case. This could be solved by clearly explaining what kind of data can be expected from a (cancer) registry. But this applies to EHR data (and other data) as well: they will not contain all health data of the patient in the observation period, only the health data related to diagnoses and treatment in the hospital.
- This may imply that the cancer registry contains other health data of the patient from before the cancer diagnosis. Here the same applies as in the previous point, i.e. we need to explain clearly what kind of data to expect from a (cancer) registry.

**3. The incidence date minus a fixed time period, for example 1 year**

_pros:_

- If 2. is not an acceptable option then this solves the issue with the choice of 1.

_cons:_

- What is a good fixed time period? 0.5 years, 1 year, ...? This seems arbitrary and could depend on the study. 

**OBSERVATION_PERIOD.observation_period_end_date**

**1. The date of the last entry for the patient in the OMOP-CDM for the particular cancer diagnosis**

_pros:_

- Cancer registries typically collect data around the diagnosis of the patient and maybe about the first line of treatment. A natural end date then is the last date related to the diagnosis or the first line of treatment.

_cons:_

- Many cancer registries also collect information about the death of the patient. This should then be in a separate observation period? Would this lead to issues in studies?
- If the patient develops a different cancer at a later time then this is a separate observation period. Does this lead to issues in studies? For example if you want to find out if the patient has a history of cancer?

**2. The death date/date on which the vital status of the patient was last checked**

_pros:_

- If option 2 is chosen for the start date then this is a logical end date.

_cons:_

- It implies that there is information about the disease between diagnosis/first line of treatment and death. For example about recurrence, second line of treatment, etc. This is usually not the case in cancer registries. This could also be solved by explaining clearly what can and what cannot be expected from a cancer registry. 

**(Current) choice for Netherlands Cancer Registry (NCR)**

For the NCR we have chosen option 2 for the start date (date of birth or 1989 (date from which we have national coverage)) and option 2 for the end date (death date/date on which we last checked the vital status of the patient (which we do once a year)). 

**Our reasoning is:**

- We follow patients their whole life and when they develop a cancer the relevant data (i.e. relevant for a cancer registry) is entered in the cancer registry. This solves the issue mentioned in option 1 of the start date.
- It seems a more natural choice for people with multiple primary diagnoses. We capture all of those so it should be one observation period. Multiple observation periods would imply that we may not register all of them.

**Potential issues with this choice:**

- We do not collect all information about the patient during the observation period (only primary diagnosis and first line of treatment). We think this is not an issue that should be solved with the observation period but it is a characteristic of the data source (i.e. a cancer registry).
- Emigration and immigration: because of this we may not actually capture all primary cancers of a patient even though the observation period implies we do. For a country like The Netherlands this would lead to only a minor error and it only seems relevant for the question "does the patient have a history of cancer?". None of the other solutions solve this problem: In all solutions, if you find a previous diagnosis for a certain patient then you can say with certainty that the patient has a history of cancer. In our solution, if you don't find a previous diagnosis, you can be fairly sure the patient does not have a history of cancer (after 1989) but there is a small chance that you're wrong. In the other solutions you can't say anything. 
- It may imply that the registry contains all people in the region (not just the ones that were diagnosed with cancer). We think this should also be a characteristic of the data source, not something that needs to be solved in the observation period.

**We would like to know:**
- What do you think is the best option (and why)?
- Do you have a solution that we hadn't thought of?

**Comment by CotterpinDoozer** [comment]
We have discussed back and forth in the Cancer Registry of Norway (CRN) as well. 

We started out by choosing alternative 1 for start date (date of incidence), and had end date to be either date of death or emigration or - if the patient was still alive - date of extraction of the OMOP-data from our cancer registry database. 

After the discussions @peterprinsen-iknl refer to in his post, and after having discussed back and forth in the CRN, we have ended up changing our view. This is based on several things, but one of the most important was the sentence here: http://cdmqueries.omop.org/observation-period (my bolding of text): 

> An Observation Period is the span of time when **a Person** is expected to have the potential of Drug and Condition information recorded.

So, since this is based on person, not case, we agreed in the CRN that: 
**OBSERVATION_PERIOD.observation_period_start_date = The date of birth or the date from which there is full coverage of the region by the registry**

and, 

**OBSERVATION_PERIOD.observation_period_end_date = The death date/date on which the vital status of the patient was last checked** For pasients still alive, "last checked" is equal to the extraction date for the current version of the dataset. 

This is the time-span where we can observe the cancer-related information for a patient, including for instance also prior screening participations etc. which happens before the incidence date/date of diagnosis.

**Comment by cgreich** [comment]
Friends:

This is an old discussion, but in a cancer registry it is even more pertinent. 

> We do not collect all information about the patient during the observation period (only primary diagnosis and first line of treatment).

That is the problem. The Observation period is like a surveillance camera. It should give you a reasonable expectation that if something happens it will be in the data. So, the meaning of the **absence of a record** matters here. In a cancer registry, that would be cancer diagnoses and treatments and the dynamics of the diseases (progression, remission, symptoms, toxicity, death, etc.). But still. If you cannot expect any meaningful data in a period of time it should not be part of the Observation Period. 

From that perspective, I wouldn't put the start at the birth date or 1989. It should be the date when data are showing up in the registry. If that is diagnosis date - so be it. If studies want a washout period - too bad. Just pre-dating the start date doesn't help you.

> Multiple observation periods 

Should not be a problem. The standard methods and tools take care of that.

**Reply by peterprinsen-iknl** [reply]
@cgreich , thank you for your reply. @golozara, I am wondering what your view is on this (in light of the PIONEER study-a-thons). I see Christian's point and I am not opposed to adopting it. But for us that means we couldn't have participated in the PIONEER study-a-thons. Or will this washout period not be essential anymore in the future if we adopt the Date of Initial Diagnosis (which we haven't yet but we will implement soon)?

**Comment by golozara** [comment]
@peterprinsen-iknl: That is one of the common limitation we are dealing with when it comes to using cancer registry data for these kind of population-based network studies. On one hand, the granular information in registries make them unique sources of information especially when it comes to tumor characteristics at baseline and survival, and on the other hand, the cross sectional nature of the data and lack of additional information required for phenotype development and analyses limit their utility. Unfortunately, changing the definition have huge impact on the validity and reliability of the cohorts in other datasources. We also cannot change the definitions in one databases because that would be against the our notion of standardized cohorts and analytics across the network.  My suggestion is to get together and identify objectives (from the studyathons and other studies) that can be addressed in NCR and only use NCR for those instances. I know there is still a good proportion of the ongoing pioneer studies that can be addressed in NCR. We should also investigate how the data can be augmented through linkage with other data sources.
